### PR TITLE
#1753 Add ignore white spaces option to csv read

### LINKED ISSUE #1753: Add trimming whitespaces feature to CSV reader
Add trimming whitespaces feature to CSV reader

```
.option("ignoreLeadingWhiteSpace", "true/false")
.option("ignoreTrailingWhiteSpace", "true/false")
```

### PR DESCRIPTION
**Pull request by Zejnilovic**
RN: Add an option to ignore trailing and leading white spaces on CSV read

Bash scripts enhancements to come

Closes #1753

### CODE REVIEW

File: spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/config/StandardizationConfigParser.scala [hunk 1]
```diff
@@ -199,7 +211,9 @@ object StandardizationConfigParser {
         config.csvDelimiter.map(_ => unsupportedOptionError("--delimiter", csvFormatName)),
         config.csvEscape.map(_ => unsupportedOptionError("--escape", csvFormatName)),
         config.csvHeader.map(_ => unsupportedOptionError("--header", csvFormatName)),
-        config.csvQuote.map(_ => unsupportedOptionError("--quote", csvFormatName))
+        config.csvQuote.map(_ => unsupportedOptionError("--quote", csvFormatName)),
```
Comment (dk1844) [inline spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/config/StandardizationConfigParser.scala:214]: `--csv-ignore-leading-white-space` to be consistent?

Comment (Zejnilovic) [reply spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/config/StandardizationConfigParser.scala:214]: Yes. Done. Funnily the executable one and in tests, I have `ignore`.

**Review by dk1844 — APPROVED** [review]
LGTM (just read the code)

**Review by HuvarVer — APPROVED** [review]
- [ ] reviewd
- [ ] build
- [ ] run
- [ ] tested

Tested run std whit white space by new attributes set to true and false and without these attributes. Made quick smoke test on UI nad backend (create dataset, run std and conf).

**Pull request merged**